test/CadlRanchProjects/server/path/multiple/src/Generated/tspCodeModel.json

```diff
@@ -20,7 +20,7 @@
    ],
    "Namespace": "Server.Path.Multiple",
    "Description": "Service versions",
-   "IsExtensible": true,
+   "IsExtensible": false,
```

Comment:
could you confirm whether this change is caused by some tsp file is not updated from enum to Union?
If that is the case, shall we update the cadl-ranch case directly?

Reply:
This is API version. Compiler only allows `enum` to define an API version. I don't think we should change cadl-ranch. And it's OK for us to make it `IsExtensible` false, because we handle it in a separate way.